File: Luis Mera/java/Leccion04/SobreEscritura/src/domain/Empleado.java
package domain;

public class Empleado {
    protected String nombre;
    protected double sueldo;

    public Empleado(String nombre, double sueldo) {
        this.nombre = nombre;
        this.sueldo = sueldo;
    }

    public String obtenerDetalles() {
        return "Nombre: " + nombre + ", Sueldo: " + sueldo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: GianniPasquinelli/phyton/leccion13/Empleado.py
class Empleado:
  def __init__(self, nombre, salario):
    self.nombre = nombre
    self.salario = salario
    
  def __str__(self):
    return self.nombre + ' ' + str(self.salario)
  
  def mostrar_detalles(self):
    return self.__str__()